tests/testthat/test-download_models.R

```diff
@@ -0,0 +1,55 @@
+
+# setup
+temp_dir <- tempdir()
+save_models_dir <- file.path(temp_dir, "save_mods")
+# create directories and use on exit statements to ensure cleanup at the end.
+dir.create(save_models_dir)
+on.exit(unlink(save_models_dir, recursive = TRUE), add = TRUE)
+if (!dir.exists("inst")) {
+  dir.create("inst")
+  dir.create("inst/extdata")
+  on.exit(unlink("inst", recursive = TRUE), add = TRUE)
+}
+if (!dir.exists("inst/extdata")) {
+    dir.create("inst/extdata")
+    on.exit(unlink("inst/extdata", recursive = TRUE), add = TRUE)
+}
+
+test_that("download_models with defaults work", {
+    download_models()
+    expect_true(file.exists("inst/extdata/models"))
+    expect_true(file.exists("inst/extdata/models/Simple"))
+    expect_true(file.exists("inst/extdata/models/Simple/ss.par"))
+})
+
+test_that("download_models with different dir works", {
+    download_models(dir = save_models_dir)
+    expect_true(file.exists(file.path(save_models_dir, "models")))
+    expect_true(file.exists(file.path(save_models_dir, "models", "Simple")))
+    expect_true(file.exists(file.path(save_models_dir, "models", "Simple", "ss.par")))
+})
+
+# test_that("download models works with different branch", {
```

Comment:
Should we delete this rather than leave in an unused test? I am guessing it is commented out because branches other than main will be transient?

Reply:
Yes, all other branches are feature branches, so I'll just delete the test.